core/appmodule/module.go

```diff
@@ -71,10 +80,3 @@ type HasEndBlocker interface {
 	// a block.
 	EndBlock(context.Context) error
 }
-
-// UpgradeModule is the extension interface that upgrade module should implement to differentiate
```

Comment:
Do you mean keep `UpgradeModule` for future use with `PreBlock`?

Reply:
We are not going to include this PR in v0.50 because it is too late in the release (the plan is to get a rc out today).
And after RC there are no more breaking changes.
We are investigating if we should keep or revert the `RunMigrationBeginBlock` changes in v0.50.

Reply:
Sorry for the back and forth.

Reply:
Okay, as discussed on Slack, we can keep the deletion then :D